noxfile.py

```diff
@@ -473,6 +473,9 @@ def test_jupyter_notebooks(session):
         )
 
     session.install("jupyter", "nbval", "pyzmq")
+    if platform.system() == "Windows":
+        session.install("pywin32==225")
```

Comment:
Now that you figured out what works, maybe try to move the installation back to noxfile?

```
conda create -n hydra python=<< parameters.py_version >> pywin32==225 virtualenv==20.0.33 -qy
```

Reply:
Hmm... I'm getting a 'package not found' error from conda (commit https://github.com/facebookresearch/hydra/pull/1650/commits/be5602bb470480bba78fcb76b7bc24acc37739b9).

Reply:
And using pip in the `.circleci/config.yml` file is not working either (https://github.com/facebookresearch/hydra/pull/1650/commits/44427844930aeff949178e15cb1798bfa2fff3ec).